extensions/binding/org.eclipse.smarthome.binding.digitalstrom/README.md

```diff
@@ -203,54 +365,170 @@ All devices support some of the following channels:
+      Text item=TotalElectricMeter 
+    }
+  
+   Frame label="digitalSTROM-Meter"{
+      Text item=TotalActivePowerDsm
+      Text item=TotalElectricMeterDsm
+    }
+  }
+  
+  Frame label="Climate" {
+   Frame label="heating/cooling"{
+      Switch item=SensorSwitch 
+    }
+    
+   Frame label="iSens200"{
+      Text item=TempIndoor
+      Text item=HumidityIndoor
+    }
+    
+   Frame label="Target temperature"{
+      Slider item=Temperature
+      Text item=Temperature
+    }
+  }
+  
+  Frame label="Shade"{
+   Frame label="Rollerschutter"{
+      Slider item=Shade 
+      Text item=Shade
     }
-    Frame {
-        Text item=TotalActivePower 
-        Text item=TotalElectricMeter 
+    
+   Frame label="Blind"{
+      Slider item=BlindPosition 
+      Slider item=BlindAngle
     }
-    Frame {
-        Switch item=Scene 
+  }
+  
+  Frame label="Scenes"{
+    Frame label="TV scene"{
+      Switch item=Scene 
     }
-    Frame {
-        Slider item=Shade 
+  }
+  
+  Frame label="HiFi" {
+    Frame label="TV light"{
+      Slider item=Brightness 
+      Switch item=Brightness
+      Text item=ActivePower 
+      Text item=OutputCurrent 
+      Text item=ElectricMeter 
     }
+    
+   Frame label="TV"{
+      Switch item=DeviceSwitch
+    }  
+  }
 }
 ```
+
+## General-Informations
+
+### digitalSTROM-Scenes
+
+The device scene configuration will be saved persistently to the thing properties, if the thing is not textual configured (because textual configured things will not be persisted), to update the device state faster. For that each scene configuration of each device has to read out first, because of the digitalSTROM-rule 9 that requires a waiting period of one minute, that take some time so that at the first start a scene call can be take some time to read it out and update the device state. To read it out faster only the discovered or called scenes will be red out. 
+ 
+**Note:**
+Because the digitalSTROM-Server can't inform the binding about save scene events at this time, the persistently saved scene configurations can't be updated. The current troubleshooting to read out the new scene configuration after a save scene action at the digitalSTROM-Server is the following:
+
+1. delete the thing to delete the persistently saved scene configuration
+2. restart the server to delete the temporary saved scene configuration 
+3. add the thing again to red out the scene configuration again.   
+
+### Initial state of digitalSTROM-Scenes and devices
+
+To get the device and scene state after a server start or restart, the binding uses the last called group scenes of digitalSTROM. Because of that there are two things to be observed:
+
+1. If a device status has changed through a device scene or a directly set output value, the status is maybe not correct.
+2. If the last called group scene does not read out yet, it can takes some time till the status will be updated. 
+
+### Textual configuration notice
```

Comment:
That's because of the special scene-concept of digitalSTROM. And it's currently the best way we think.
Without this “feature”, all relevant scenes have to be read out after a server reboot and this is subject to a rule that says that "only 1 reading job per minute per circuit is allowed.". So that can be take much time.

It is a bit complicated to explain it complete.